docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
+{% endhint %}
 
-### Search and Filter Resources
+Based on the schema provided in the catalog framework, you can add relevant details for each cluster. Refer [Catalog Framework](./global-configurations/catalog-framework.md) for more details. 
 
-You can search and filter resources by specific resource Kinds. You can also preview `Manifest`, `Events`, `Logs`, access `Terminal` by selecting ellipsis on the specific resource or `Delete` a specific resource.
+### Readme
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/select-resource.jpg)
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to edit the readme file.
+{% endhint %}
 
-#### Manifest
+You can also include additional information about your cluster using the Markdown editor.
 
+---
 
-A manifest is a YAML file that describes each component or resource of your deployment and the state you want your cluster to be in once applied. Once you deploy, you can always edit your manifest file. A manifest specifies the desired state of an object that Kubernetes will maintain when you apply the manifest.
+## Discovering Resources 
 
-#### Events
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have [access to the cluster](./global-configurations/authorization/user-access.md#kubernetes-resources-permissions) to discover resources.
+{% endhint %}
 
-An event is automatically generated in response to changes with other resources—like nodes, pods, or containers. For example, phases across a pod’s lifecycle transition from pending to running, or statuses like successful or failed may trigger a K8s event. The same goes for re-allocations and scheduling. These events are available until 15 minutes of deployment of the application.
+### Search and Filter
 
-#### Logs
+You can use the searchbox to browse the resources.
 
-Logs contain the logs of the Pods and Containers deployed which you can use for the process of debugging.
+![Figure 4: Locate Resources using Searchbox](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/discover-resource.gif)
 
+Moreover, you can use filters that allow you to quickly filter your workload as per labels, field selectors, or [CEL expression](https://kubernetes.io/docs/reference/using-api/cel/) as shown below.
 
-### Create Kubernetes Resource
+{% embed url="https://www.youtube.com/watch?v=E-V-ELCXtfs" caption="Filtering Workloads in Devtron" %}
 
-**Note**: As a pre-requisite, you must have a basic understanding of Kubernetes Cluster, Resources, Kinds.
+### Edit Manifest 
 
-You can create a Kubernetes resource by passing definition YAML file. You can create more than one resource by separating the resource YAMLs by ‘---’.
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to edit a manifest.
```

Comment:
I think an admin of a resource should be able to edit the manifest @uxarya-d ?

Reply:
You are right, @abhibhaw. 
Now I checked with editing a label as an admin, and it worked. 

Initially, my inference was based on this error which led me to believe it cannot be edited by anyone other than superadmin (gif shared below). 
![409 error](https://github.com/devtron-labs/devtron/assets/141001279/c1398980-1156-406a-bc7d-a75e864f19ae)